theme/theme.ts

```diff
@@ -7,12 +7,15 @@ export type Theme = 'blue' | 'teal' | 'green' | 'indigo' | 'orange' | 'purple' |
 export const THEMES: Theme[] = ['blue', 'green', 'teal', 'indigo', 'orange', 'purple', 'gray'];
 
 export const TextType = {
-  h1: `font-sniglet text-4xl font-extrabold`,
-  h2: `font-sniglet text-2xl font-extrabold`,
-  h3: `font-sniglet text-xl`,
-  base: `font-roboto text-base`,
-  small: `font-roboto text-sm`,
-};
+  '3xl': `font-sniglet text-6xl font-extrabold`,
+  '2.5xl': `font-sniglet text-5xl font-extrabold`,
+  '2xl': `font-sniglet text-4xl font-extrabold`,
+  '1.5xl': `font-sniglet text-3xl font-extrabold`,
+  xl: `font-sniglet text-2xl font-extrabold`,
+  lg: `font-sniglet text-xl`,
+  md: `font-roboto text-base`,
+  sm: `font-roboto text-sm`,
+} as const;
```

Comment:
```suggestion
  6xl: `font-sniglet text-6xl font-extrabold`,
  5xl: `font-sniglet text-5xl font-extrabold`,
  4xl: `font-sniglet text-4xl font-extrabold`,
  3xl: `font-sniglet text-3xl font-extrabold`,
  2xl: `font-sniglet text-2xl font-extrabold`,
  xl: `font-sniglet text-xl`,
  base: `font-roboto text-base`,
  sm: `font-roboto text-sm`,
} as const;
```
@prateek3255 I'm a little uncertain about this. The key names should ideally match the tailwind's text font size class name.

why did we decide to have 1.5xl? 🤔

Reply:
Yeah, the initial naming was like that, and when I started refactoring, I went along with the initial naming and ended up with this 😓. I'll fix this ✅

Reply:
Fixed 👍🏼